src/vmm/src/builder.rs

```diff
@@ -1038,7 +1042,12 @@ pub mod tests {
     }
 
     fn default_mmio_device_manager() -> MMIODeviceManager {
-        MMIODeviceManager::new(arch::MMIO_MEM_START, (arch::IRQ_BASE, arch::IRQ_MAX))
+        MMIODeviceManager::new(
```

Comment:
You would need to add a comment here stating that the unwrap is safe since memory/irq ranges are in order and non-overlapping.

Reply:
nevermind, it's in the test module.